src/core/utils/helpers/general.ts

```diff
@@ -555,6 +555,10 @@ export const formatDanishPhoneNumber = (phoneNumber: string) => {
 export const parseBoolean = (value?: string): boolean =>
   value?.toLowerCase() === "true";
 
+export const isActivationKeys = (key: string): boolean => {
```

Comment:
Would it be possible to name it more precisely?
Either the context could be thrown into the name, eg.: "isModalActivationKeys"
Or a bit clumsy, yet understandable, alternative could be: "keysEnterOrSpaceWasPressed"
Or you could make a function with the previous name suggested: "keysEnterOrSpaceWasPressed" and then make a function for each purpose that points at "keysEnterOrSpaceWasPressed" - example:
```typescript
// Global helper
export const keysEnterOrSpaceWasPressed = (key: string): boolean => {
  return key === "Enter" || key === " ";
};

// Local function
export const materialModalIsOpen = (key: string): boolean => keysEnterOrSpaceWasPressed(key)
```